nixos/modules/services/databases/mongodb.nix

```diff
@@ -12,13 +12,11 @@ let
 
   mongoCnf = pkgs.writeText "mongodb.conf"
```

Comment:
Upstream [documentation](https://docs.mongodb.com/manual/reference/configuration-options/#use-the-configuration-file) uses `.conf` files.
Do you think we should change to `.yaml` anyway?